x/examples/website-wrapper-app/wrapper_app_project/template/android/app/src/main/java/org/getoutline/pwa/Config.kt.handlebars

```diff
@@ -1,4 +1,4 @@
-package org.getoutline.pwa
+package {{appId}}.android
```

Comment:
Do we really need the `.android` postfix? The original package is `org.getoutline.pwa`, without `.android`.